### PR TITLE
updates on partners/payment-agent

### PR DESCRIPTION
**Pull request by alamin-deriv**
Changes:

-   Added the new P2P banner in `partners/payment-agent` page

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [x] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/partners/payment-agent/_p2p_banner.tsx [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+    line-height: 40px;
+    color: var(--color-white);
+    margin-bottom: 16px;
+
+    @media ${device.tabletL} {
+        margin-bottom: 8px;
+        max-width: 378px;
+    }
+`
+const StyledText = styled(Text)`
```
[the remark this answers is not in the document]
Comment (alamin-deriv) [reply src/pages/partners/payment-agent/_p2p_banner.tsx:102]: I tried changing it to what you suggested but didn't work it just become bold the `Header` and nothing change

File: src/pages/partners/payment-agent/_p2p_banner.tsx [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+    font-weight: bold;
+    border-radius: 4px;
+    width: 146px;
+    white-space: nowrap;
+
+    @media ${device.tabletL} {
+        height: fit-content;
+        padding: 6px;
+    }
+`
```
Comment (sean-binary) [inline src/pages/partners/payment-agent/_p2p_banner.tsx:136]: Is this any different than the original button component?

File: src/pages/partners/payment-agent/_who-can-apply.tsx [hunk 3]
```diff
@@ -70,6 +70,17 @@ const ButtonWrapper = styled(CenteredSection)`
...
+const StyledListHeader = styled(Header)`
+    width: 100%;
+    max-width: 42.8rem;
+    margin-top: 4rem;
+
+    @media ${device.tabletL} {
+        font-size: 14px;
+        line-height: 20px;
+    }
+`
```
Comment (sean-binary) [inline src/pages/partners/payment-agent/_who-can-apply.tsx:82]: Widths, max widths, margins should be set on its parent wrapper instead. Font size and line height adjust for responsive shouldn't be needed if the correct `type` is used.

File: src/pages/partners/payment-agent/_who-can-apply.tsx [hunk 4]
```diff
@@ -128,6 +139,25 @@ const WhoCanApply = () => {
                                 text={item.text}
                             />
                         ))}
+                        <StyledListHeader as="h4" size="1.6rem">
```
Comment (sean-binary) [inline src/pages/partners/payment-agent/_who-can-apply.tsx:142]: `size` should not be set but instead use `type`.

File: src/pages/partners/payment-agent/_p2p_banner.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (sean-binary) [reply src/pages/partners/payment-agent/_p2p_banner.tsx:102]: Let's discuss to see why that's the case. Please reach out when you are available.

**Review by sean-binary — CHANGES_REQUESTED** [review]

File: src/pages/partners/payment-agent/_p2p_banner.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (alamin-deriv) [reply src/pages/partners/payment-agent/_p2p_banner.tsx:102]: No, don't worry, I saw the issue

**Comment by alamin-deriv** [comment]
@NikitaK-deriv has taken over this task and he has a new [PR](https://github.com/binary-com/deriv-com/pull/2670)

**Pull request closed without merging**